src/form.js

```diff
@@ -211,7 +211,7 @@ define([
             _this.fireChange(e.mug);
         });
         this.instanceMetadata = [InstanceMetadata({})];
-        this.knownInstances = {}; // {<instance id>: <instance src>}
+        this.knownInstances = {}; // {<instance id>: <instance src or children>}
```

Comment:
Ugh, I just realized this will probably break copy/paste for xpath expressions that reference data instances. See [xpath serialization in mugs.js](https://github.com/dimagi/Vellum/blob/manual-instance/src/mugs.js#L705-L718) as well as [`parseInstanceRefs` in form.js](https://github.com/dimagi/Vellum/blob/manual-instance/src/form.js#L381). Might want to add a test for that.